wolfcrypt/src/asn.c

```diff
@@ -174,10 +174,16 @@ WOLFSSL_LOCAL int GetLength_ex(const byte* input, word32* inOutIdx, int* len,
             return BUFFER_E;
         }
 
+        if (bytes > sizeof(int)) {
```

Comment:
sizeof(int) isn't portable.  What are you checking, 2, 4?

sizeof(length) is easier to understand.

Reply:
Fixed.
Making sure that the number of length bytes won't overflow the length variable.